dev/src/index.ts

```diff
@@ -703,14 +703,13 @@ export class Firestore {
    *   console.log(`Second document: ${JSON.stringify(docs[1])}`);
    * });
    */
-  getAll(
-      documentRef: DocumentReference,
-      ...moreDocumentRefsOrReadOptions: Array<DocumentReference|ReadOptions>):
-      Promise<DocumentSnapshot[]> {
+  getAll(...documentRefsOrReadOptions: [
+    DocumentReference, ...Array<DocumentReference|ReadOptions>
+  ]): Promise<DocumentSnapshot[]> {
```

Comment:
I'm a little surprised that this behaves any differently than the original code.  In fact, I am somewhat suspicious.  I just noticed that our tests aren't actually being properly type-checked (maybe because we haven't landed https://github.com/googleapis/nodejs-firestore/pull/512 yet).  So I'm not sure your tests are actually validating anything. 😢 

Would it be possible for you to validate this change in your original project / app where you hit the destructuring problem?  (e.g. by manually applying the fix to the .d.ts file)

Reply:
Just validated, you're right, doesn't work as intended when I manually applied the fix on my app. I was too trusting of VSCode to type check, wasn't aware of #512, sorry. Will push the updated PR with the single argument.